src/contents/ui/main.qml

```diff
@@ -44,13 +44,39 @@ Item {
                 Layout.rightMargin: PlasmaCore.Units.smallSpacing * 2
             }
 
+            Item {
+                visible: plasmoid.configuration.separateText
+                Layout.preferredHeight: column.implicitHeight
+                Layout.preferredWidth: column.implicitWidth
+                    
+                ColumnLayout {
+                    id: column
+                    spacing: 0
+                    anchors.fill: parent
+                    ScrollingText {
+                        overflowBehaviour: plasmoid.configuration.textScrollingBehaviour
+                        font.bold: true
+                        speed: plasmoid.configuration.textScrollingSpeed
+                        maxWidth: plasmoid.configuration.maxSongWidthInPanel * units.devicePixelRatio
+                        text: player.title
+                    }   
```

Comment:
```suggestion
                    }
```
Can you remove this and the other trailing spaces? If you use vs-code you can highlight them with `Trailing Spaces` plugin

Reply:
Done. If you would like to have anything else changed just let me know :)

Also happy new year 🎉